src/generate/renders/CliReport.ts

```diff
@@ -69,7 +69,7 @@ export class CliReport extends TemplateBase {
                     converter,
                 ),
             ],
-            ['azExample', new RenderInput(['commandStringItems'], {})],
```

Comment:
Is this PR going to refactor report.md generation logic into template based ?

Reply:
no, this PR is for add max&min mock test only